src/vm_pmu.rs

```diff
@@ -365,7 +368,110 @@ impl VmPmuState {
         }
     }
 
-    /// Calls the SBI configure_matching_counters() and performs internal bookkeeping on counter state.
+    fn configure_delegated_matching_counters(
+        &mut self,
+        counter_index: u64,
+        counter_mask: u64,
+        config_flags: PmuCounterConfigFlags,
+        event_type: PmuEventType,
+        event_data: u64,
+    ) -> SbiResult<u64> {
+        use sbi::PmuHardware::*;
+        fn configure_counter_csrs(
+            config_flags: PmuCounterConfigFlags,
+            event_type: PmuEventType,
+            event_data: u64,
+            counter_index: u64,
+        ) {
+            let mut shpmevent = LocalRegisterCopy::<u64, shpmevent::Register>::new(0);
+            // Ignore vuinh from the VM and translate uinh/sinh
```

Comment:
We shouldn't ignore vuinh/vsinh as we are already running a TVM  :).

Reply:
Changed....PTAL.